jdi-light-applitools-tests/src/test/java/com/jdi/data/DefaultDataProvider.java

```diff
@@ -0,0 +1,23 @@
+package com.jdi.test.data;
+
+/**
+ * Created by Roman Iovlev on 10.11.2018
+ * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+
+import jdisite.entities.ContactInfo;
+
+public class DefaultDataProvider {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'DefaultDataProvider' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5478709)